lib/xgu/xgux.h

```diff
@@ -13,11 +13,11 @@ void xgux_draw_arrays(XguPrimitiveType mode, unsigned int start, unsigned int co
 }
 
 inline
-void xgux_set_clear_rect(uint32_t x1, uint32_t x2,
-                         uint32_t y1, uint32_t y2) {
+void xgux_set_clear_rect(uint32_t left, uint32_t right,
+                         uint32_t top, uint32_t bottom) {
```

Comment:
Note: this could potentially do what GL does: `(GLint x, GLint y, GLsizei width, GLsizei height);` [but `unsigned int`] - it clarifies wether values are inclusive and how the coordinate system works.

Reply:
Fixed. :+1: